tests/bsim/bluetooth/samples/battery_service/src/central_test.c

```diff
@@ -399,7 +444,31 @@ static void subscribe_battery_level(const struct bt_gatt_attr *attr)
 	} else {
 		LOG_DBG("Battery level [SUBSCRIBED]\n");
 	}
-	read_battery_level(attr);
+	bt_gatt_read(default_conn, &read_blvl_params);
+}
+
+static void subscribe_battery_critical_status(const struct bt_gatt_attr *attr)
+{
+	int err;
+
+	battery_critical_status_sub_params = (struct bt_gatt_subscribe_params){
+		/* In Zephyr, it is common practice for the CCC handle
+		 * to be positioned two handles after the characteristic handle.
+		 */
+		.ccc_handle = bt_gatt_attr_get_handle(attr) + 2,
```

Comment:
Just beware that this is undocumented any may fail at any time. 

The better solution is of course to do a proper discovery of it, or use the auto CCC discover feature